.travis.yml

```diff
@@ -72,4 +40,58 @@ script:
   - export XTENSOR_WITH_TBB=0
   - export XTENSOR_WITH_XSIMD=0
   - python setup.py build_ext --with-tests
-  - python setup.py test -v
\ No newline at end of file
+  - python setup.py test -v
+
+jobs:
+  include:
+    - stage: singularity-build-push
+      if: branch = master OR tag IS present
+      language: go
+      go: "1.13"
+      git:
+        submodules: false
+        depth: 1
+      addons:
+        apt:
+          packages:
+            - flawfinder
+            - squashfs-tools
+            - uuid-dev
+            - libuuid1
+            - libffi-dev
+            - libssl-dev
+            - libssl1.0.0
+            - libarchive-dev
+            - libgpgme11-dev
+            - libseccomp-dev
+        homebrew:
+          packages:
+            - squashfs
+          update: true
+      install:
+        - SINGULARITY_BASE="${GOPATH}/src/github.com/sylabs/singularity"
+        - export PATH="${GOPATH}/bin:${PATH}"
+        - mkdir -p "${GOPATH}/src/github.com/sylabs"
+        - cd "${GOPATH}/src/github.com/sylabs"
+        - git clone -b v3.5.0 https://github.com/sylabs/singularity
+        - cd singularity
+        - ./mconfig -v -p /usr/local
+        - make -j `nproc 2>/dev/null || echo 1` -C ./builddir all
+        - sudo make -C ./builddir install
+      before_script:
+          #  token used for push commnad
+        - echo -e "$SYLABS_TOKEN" > token
+        - singularity remote login --tokenfile ./token || exit 1
+          #  key used for image signing
+          #  for travis ci pgp key block you must replace newline with '\\n', and replace spaces with '\ '
+        - echo -e "$SINGULARITY_KEY" > skey
+        - head -n 4 ./skey
+        - echo $SINGULARITY_KEY_PW | singularity key import ./skey || exit 1
+      script:
+        - cd $TRAVIS_BUILD_DIR
+        - export SINGULARITYENV_PYTHON_VERSION="3.7.5"
```

Comment:
Why do we have a fixed Python version here?

Reply:
I can remove that and use `PYTHON_VERSION` instead, that way it'll take the first option from `env.jobs.PYTHON_VERSION`, which would be `PYTHON_VERSION="3.7.5"`.

So the version of python in the image will always be equal to the first-defined version of python in the test matrix.

Reply:
Using `PYTHON_VERSION` looks better to me.